### PR TITLE
jquery and DOM diagnostic

### PR DESCRIPTION
**Pull request by shireen-bean**
all questions answered, commit one

### CODE REVIEW

**Comment by shireen-bean** [comment]
comfort 4
clarity 5

File: diagnostic.md [hunk 1]
```diff
@@ -4,7 +4,10 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+div
+h1
+ul
```
Comment (berziiii) [inline diagnostic.md:9]: Correct.

File: diagnostic.md [hunk 2]
```diff
@@ -14,6 +17,14 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
...
 
 <!-- your answer starts here -->
+$('OL').find('li')
+
+or you could go all the way to the top to ensure that OL is really the parents and add
+.parent() befoe 
+
+or
+let allListItems = $('li');
+$('OL').find(allListItems)
```
Comment (berziiii) [inline diagnostic.md:27]: Correct.

File: diagnostic.md [hunk 3]
```diff
@@ -23,7 +34,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('.listerItem').parent().find('form')
```
Comment (berziiii) [inline diagnostic.md:37]: Correct.

File: diagnostic.md [hunk 4]
```diff
@@ -33,6 +44,14 @@ methods.
 
 <!-- your answer starts here -->
 
+setter:
+$('#ajax-error-message').text('New Error')
+$('#ajax-error-message').css('color',red)
+
+getter:
+$('#ajax-error-message').html()
+$('#ajax-input').val();
```
Comment (berziiii) [inline diagnostic.md:53]: Correct.

File: diagnostic.md [hunk 5]
```diff
@@ -41,6 +60,8 @@ In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
 
+Virtual representation of what the browser interperates the combination of html, css, js etc files as.  It is like a network of piping that fills with data/text/styles when it is told to.
```
Comment (berziiii) [inline diagnostic.md:63]: Correct.

File: diagnostic.md [hunk 6]
```diff
@@ -49,6 +70,8 @@ Please give two examples of things that are commonly confused for the DOM.
 
 <!-- your answer starts here -->
 
+The DOM is neither a source nor a rendered page.
```
Comment (berziiii) [inline diagnostic.md:73]: Correct.

File: diagnostic.md [hunk 7]
```diff
@@ -58,6 +81,8 @@ item elements) to the body DOM node, then hides it.
 
 <!-- your answer starts here -->
 
+$('li').parent().parent().hide();
```
Comment (berziiii) [inline diagnostic.md:84]: Correct.

File: diagnostic.md [hunk 8]
```diff
@@ -66,4 +91,6 @@ Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
 
+$('div').on('click',clickHandler);
```
Comment (berziiii) [inline diagnostic.md:94]: Correct.

**Comment by berziiii** [comment]
8 / 8. Nice work!

**Pull request closed without merging**